torch/csrc/autograd/python_function.cpp

```diff
@@ -806,6 +806,12 @@ PyObject* THPFunction_name(PyObject* self, PyObject* noargs) {
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPFunction_clear_saved(PyObject* self, PyObject* noargs) {
+  HANDLE_TH_ERRORS((THPFunction*)self)->saved_variables.clear();
```

Comment:
Also use `((THPFunction*)self)->cdata->release_variables()` which does the right logic at the Node level.

Reply:
I don't actually see how to use this function? It seems like `release_variables` is on PyNode, not `THPFunction*`. How do I get a PyNode from a `THPFunction*`?